create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -476,6 +503,10 @@ private ArrayNode createPublicationChannelsResponseWithJournalUri(URI journalUri
                 .collect(Collectors.toList());
     }
 
+    private String keepOnlyTheEid(){
+        return scopusData.getDocument().getMeta().getEid();
+    }
```

Comment:
Merkelig metode navn siden den er bare en short-cut for getEid().

Reply:
Fixed